### ISSUE TITLE
Increased amyloid beta 40 isoform level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7073]
**Preferred term label:**
Increased amyloid beta 40 isoform level

**Synonyms**
Increased beta amyloid 40 level, Increased Abeta 40 level

**Definition (free text, please give PubMed ID)**
Higher than normal level of amyloid beta 40 found in cerebrospinal fluid.
PMID:19273758

**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta 40 isoform level 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 607822

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added
